packages/connect-thegraph/src/connector.ts

```diff
@@ -65,6 +65,12 @@ export default class ConnectorTheGraph extends GraphQLWrapper
     super(_orgSubgraphUrl, verbose)
   }
 
+  async connect() {}
```

Comment:
I wonder what kind of feature you envision we can include in this method for the organization connector.

Reply:
On #135, it is used to store the connection, as a second step: https://github.com/aragon/connect/pull/135/files?file-filters%5B%5D=.ts#diff-7b005797d9aa8e707f98f0dc25a4017fR75

Doing it in two steps is necessary because the connection object contains the connector :smile: